docs/servo_power_test/src/test_runner.ts

```diff
@@ -21,6 +23,11 @@ export class IterationData {
     this.histogramDataList = histogramDataList;
     this.graph = graph;
     this.graph.clearHistogram();
+    this.powerSum = 0;
+    histogramDataList.forEach(powerData => {
+      this.powerSum += powerData;
+    });
+    this.powerAverage = this.powerSum / histogramDataList.length;
```

Comment:
Please handle the case when the length of the data becomes 0 (since it will raise DivideByZero exception or something)

Reply:
in Typescript, when some number is divided by 0, it returns `NaN` or `Infinity`  (whose type is `number`) and it doesn't cause some error. But should I handle the case...?

Reply:
I see. Thank you for comments!